src/hooks/index.ts

```diff
@@ -648,9 +648,8 @@ export const useExchangeRate = ({
   const query = useQuery<number, Error>({
     suspense: true,
     staleTime: 60000,
-    retry: 3,
     retryDelay: 1000,
-    queryKey: [to],
+    queryKey: [wallet.id, 'exchangeRate', {to}],
```

Comment:
I think we should leverage the query cache across the wallets.

Reply:
our backend sometimes is very unstable due to the load, and often we receive some errors, perhaps a few retries could save the user from a bad experience on the first load

Reply:
what about ergo wallets?

Reply:
either way, each key should be loggable / readable

Reply:
the symbol will be different for the default asset ERG, the network won't affect the price